align/pnr/toplevel.py

```diff
@@ -88,14 +88,15 @@ def RouteWork( mode, current_node, *, metal_l=signal_routing_metal_l, metal_u=si
             logger.info("End MNA")
             #return
 
-
-        RouteWork(2, current_node, metal_l=power_grid_metal_l, metal_u=power_grid_metal_u)
+        if len(current_node.PowerNets)>0:
+            RouteWork(2, current_node, metal_l=power_grid_metal_l, metal_u=power_grid_metal_u)
```

Comment:
@Lastdayends Is it okay to just not run power routing in there are no power nets, or does this change some data structures around that is always needed even if there are no power nets.
Would it be better to move to check to within the power grid and power routing code? Make it more self contained.

Reply:
@stevenmburns 

I will modify the check into the code to keep self contained.